web/src/SidebarTriggerButton.scss

```diff
@@ -42,6 +43,15 @@ $trigger-button-active-color: $color-green;
   mask-size: 40%;
 }
 
+
+.SidebarTriggerButton.isQueued {
+  background-color: $trigger-button-color;
+  -webkit-mask: url(assets/svg/trigger-button.svg) no-repeat 50% 50%;
+  mask: url(assets/svg/trigger-button.svg) no-repeat 50% 50%;
+  animation: spin 1s linear infinite;
+  pointer-events: none; // disable click
```

Comment:
using pointer-events: none here is pretty bad for a bunch of a11y and UX reasons. you can still press the button with the keyboard for example. and it prevents tooltips from working.

instead, please mark the button disabled, and add a tooltip that explains why it's disabled.